### PR TITLE
Update plv8.md with formatting and another example

### PR DESCRIPTION
**Pull request by deinspanjer**
* Add subheadings under the Utility section to make it easier to find information about specific functions (I skimmed over find_function twice before finally remembering the name and doing a string search for it)
* Quick example of plv8.elog

### CODE REVIEW

File: doc/plv8.md [hunk 1]
```diff
@@ -459,13 +461,20 @@ PL/v8 provides the following utility built-in functions.
 - `WARNING`
 - `ERROR`
 
+```javascript
+var msg = 'world';
```
Comment (JerrySievert) [inline doc/plv8.md]: unfortunately, this won't work - we're published on pgxn, which does not support gfm.  please check how other examples are formatted.

Comment (deinspanjer) [reply doc/plv8.md]: Ahh, sorry I was only thinking about github syntax.  I'll clean that up right now.

Comment (JerrySievert) [reply doc/plv8.md]: don't worry, you're not alone :) it was @theory who put in the PR to fix the previous formatting to work correctly with pgxn.  thanks much for the update, I'll finish the review in just a little bit!

**Comment by JerrySievert** [comment]
awesome, thanks!

**Pull request merged**